duckduckgo/macos-browser
# Fix of the detached web inspector after tab switching

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203974731690101/f
CC:

**Description**:
Fix of the docked web inspector detaching itself after opening a new tab

**Steps to test this PR**:
1. Visit a website and open the web inspector (in docked state)
2. Open a new tab
3. Make sure there is no web inspector visible (in detached state)
4. Select the first tab with the website and make sure web inspector remains in the original position



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
